yang-semver/draft-ietf-netmod-yang-semver.xml

```diff
@@ -657,8 +669,8 @@ module ietf-yang-semver {
   // note.
   // RFC Ed. update the rev:revision-label to "1.0.0".
 
-  revision 2021-11-04 {
-    rev:revision-label "1.0.0-draft-ietf-netmod-yang-semver-05";
+  revision 2021-11-17 {
```

Comment:
Ah.  We don't need to rev the YANG module as we're not making changes.  Moreover, if we do rev the module, we need to change the date in the XML above.

Reply:
Good point. I was misinterpreting our own rules and thought the draft YANG module would always have to have the same version in the suffix as the containing draft version. But that's only if the YANG module in the draft actually changes in the new version of the draft.  I'll revert that back to 2021-11-04 and version-05.